anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-failover.adoc

```diff
@@ -0,0 +1,118 @@
+= Using Cross-Region Failover
+
+Anypoint MQ Connector for Mule 4 version 3.2.6 and later provides failover capability for standard queues.
+
+This feature is not enabled by default.
+Before using this feature, review xref:mq::mq-failover.adoc[].
+
+[IMPORTANT]
+When you enable failover for a queue and the primary region goes down,
+your data resides in a different region.
+
+
+To use the failover feature, do the following:
+
+* In your app:
+
+.. Upgrade Anypoint MQ Connector to version 3.2.6 or later.
+.. <<enable-failover-feature,Enable failover in your app.>>
+.. (Optional) <<configure-check-frequency,Change the check frequencies>>. 
+
+* In Anypoint MQ:
+.. Create or modify a queue and xref:mq::mq-failover.adoc#enable-failover[enable failover on the queue].
+
+[[enable-failover-feature]]
+== Enable Failover in Your App
+
+To enable the failover feature in your app, set the `cross.region.failover.feature.enabled` system property to `true`.
+
+To set this application property in Runtime Manager:
+
+. Sign in to Anypoint Platform and click *Runtime Manager* in the navigation menu.
+. Deploy a new app or change the settings for a deployed app.
+. On the *Properties* tab, specify values for the property:
++
+image::mq::mq-set-properties-failover-enabled.png["Properties tab with failover properties specified"]
+
+For more information, see: 
+
+* xref:cloudhub-2::ch2-manage-props.adoc[Changing App Behavior with Properties on CloudHub 2.0]
+* xref:cloudhub::cloudhub-manage-props.adoc[]
+
+Alternatively, set the property in *Run Configurations* in Anypoint Studio.
+For more information, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties in Mule Apps] 
+
+
+
+[[configure-check-frequency]]
+== Configure the Frequency of Fallback Queue Checks
+
+By default, Anypoint MQ Connector checks the fallback queue once per minute.
```

Comment:
The default is different for both the properties:
fallback.messages.status.check.interval.ms - default is 300,000 (i.e., 5 minutes)
primary.region.status.check.interval.ms - default is 60,000 (i.e., 1 minute)